js/editable-css.js

```diff
@@ -7,28 +7,29 @@
 
     var exampleChoiceList = document.getElementById('example-choice-list');
     var exampleChoices = exampleChoiceList.querySelectorAll('.example-choice');
-    var header = document.querySelector('header');
     var initialChoice = 0;
     var originalChoices = [];
-    var output = document.getElementById('output');
 
     /**
      * Enables and initializes the live code editor
      */
     function enableLiveEditor() {
+        var header = document.querySelector('header'),
+            output = document.getElementById('output');
```

Comment:
Nit, code style: We do not use the single `var` pattern. Please declare each variable on its own i.e.

```
var header = document.querySelector('header');
var output = document.getElementById('output');
```